gemrb/core/Scriptable/Actor.cpp

```diff
@@ -8591,7 +8591,7 @@ void Actor::SetSoundFolder(const ieVariable& soundset) const
 	}
 }
 
-std::string Actor::GetSoundFolder(int full, const ResRef& overrideSet) const
+String Actor::GetSoundFolder(int full, const ResRef& overrideSet) const
```

Comment:
shouldn't this return `path_t`? or is the name wrong?

Reply:
That's one of these case when I wasn't sure what to use. Either choice has implications for the code so (converting for Python vs. converting for path uses ...) so I choose against the bigger sigh for something.

Reply:
not really sure. does it return an actual path, or just the name of the directory?

Reply:
Depends on the game.